src/NLog/Config/LoggingConfigurationFileLoader.cs

```diff
@@ -248,6 +248,28 @@ public IEnumerable<string> GetDefaultCandidateConfigFilePaths(string fileName)
                         yield return Path.ChangeExtension(configurationFile.Replace(vshostSubStr, "."), ".nlog");
                     }
                 }
+#if NETSTANDARD && !NETSTANDARD1_3
+                else
+                {
+                    string processFilePath = ProcessIDHelper.Instance.CurrentProcessFilePath;
+                    string processDirectory = PathHelpers.TrimDirectorySeparators(Path.GetDirectoryName(processFilePath));
+                    if (!string.IsNullOrEmpty(entryAssemblyLocation) && !string.Equals(entryAssemblyLocation, processDirectory, StringComparison.OrdinalIgnoreCase))
+                    {
+                        // Handle dotnet-process loading .NET Core-assembly, or IIS-process loading website
+                        string assemblyFileName = Path.GetFileName(System.Reflection.Assembly.GetEntryAssembly().Location);
+                        if (!string.IsNullOrEmpty(assemblyFileName))
+                            yield return Path.Combine(entryAssemblyLocation, assemblyFileName + ".nlog");
+                        // Handle unpublished .NET Core Application
```

Comment:
Maybe a bit nitpicking, but I try to use those terms consistent :)

Reply:
Potato potato :)